job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/VersionNumberContext.groovy

```diff
@@ -0,0 +1,63 @@
+package javaposse.jobdsl.dsl.helpers.wrapper
+
+import javaposse.jobdsl.dsl.Context
+
+class VersionNumberContext implements Context {
+
+    String environmentVariableName = 'VERSION'
+    String versionNumberString = ''
+    String environmentPrefixVariable = ''
+    boolean skipFailedBuilds = false
+    boolean useAsBuildDisplayName = true
+    String projectStartDate = new Date()
+    String oBuildsToday = ''
+    String oBuildsThisWeek = ''
+    String oBuildsThisMonth = ''
+    String oBuildsThisYear = ''
+    String oBuildsAllTime = ''
+
+    void environmentVariableName(String environmentVariableName) {
+        this.environmentVariableName = environmentVariableName
+    }
+
+    void versionNumberString(String versionNumberString) {
+        this.versionNumberString = versionNumberString
+    }
+
+    void environmentPrefixVariable(String environmentPrefixVariable) {
+        this.environmentPrefixVariable = environmentPrefixVariable
+    }
+
+    void skipFailedBuilds(boolean skipFailedBuilds) {
```

Comment:
Please set the default parameter values to true. Further details: https://github.com/jenkinsci/job-dsl-plugin/blob/master/CONTRIBUTING.md